src/components/Dialog/Dialog.tsx

```diff
@@ -45,33 +46,35 @@ export default function Dialog(props: Props) {
     const classes = useStyles();
 
     return (
-        <div>
-            <MUIDialog
-                fullScreen
-                open={open}
-                onClose={onClose}
-                onEntered={onEntered}
-                onExit={onExit}
-                TransitionComponent={Transition}
-            >
-                <AppBar className={classes.appBar}>
-                    <Toolbar>
-                        <IconButton
-                            edge='start'
-                            color='inherit'
-                            onClick={onClose}
-                            aria-label='close'
-                        >
-                            <CloseIcon />
-                        </IconButton>
-                        <Typography variant='h6' className={classes.title}>
-                            {title}
-                        </Typography>
-                    </Toolbar>
-                </AppBar>
-                {open && children}
-            </MUIDialog>
-        </div>
+        <MUIDialog
+            fullScreen
```

Comment:
I don't remember why I had a div here previously, but was there a reason for its removal?

Reply:
Added it back, I think it was because when using it with SectionList it warned us that lists can only contain li, script, and template